### DISCUSSION TITLE
我家小猫超可爱

### DISCUSSION THREAD

**Discussion by tayiie** [share121/inter-knot#3282] [其他]
# 嘿嘿嘿，这么可爱的猫在我家喔
### 她叫杨杨，超可爱哒。
<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" ><img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >
#猫#
![IMG_20240718_181846_edit_562224149609522](https://github.com/user-attachments/assets/51e8fc43-9d1c-4b11-bd0b-e93895ea4d53)

**Comment by RaphaelHider** [comment]
很好，现在是我家的了<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by tayiie** [reply]
# 不可能，我这就回家看看
